integreat_cms/cms/forms/events/event_form.py

```diff
@@ -95,7 +96,21 @@ def clean(self):
         # make self.data mutable to allow values to be changed manually
         self.data = self.data.copy()
 
-        if cleaned_data.get("is_all_day"):
+        if (
+            cleaned_data["end_date"] - cleaned_data["start_date"]
+            > datetime.timedelta(6)
+            and not cleaned_data["is_recurring"]
+        ):
+            self.add_error(
+                "end_date",
+                forms.ValidationError(
+                    _(
+                        "Event duration longer than 7 days is not allowed for one-time event."
+                    ),
+                    code="required",
```

Comment:
```suggestion
                    code="invalid",
```
The problem is not really that no value was entered at all, but an invalid value was entered.